tests/workflows/basic_emitter_workflow/tests/test_workflow.py

```diff
@@ -120,12 +121,8 @@ def test_run_workflow__happy_path(mock_uuid4_generator, mock_datetime_now):
                     serialized_start_node_id: [str(start_node_span_id)],
                     serialized_next_node_id: [str(next_node_span_id)],
                 },
-                "node_executions_queued": {
-                    serialized_next_node_id: [],
-                },
-                "dependencies_invoked": {
-                    str(next_node_span_id): [serialized_start_node_id],
-                },
+                "node_executions_queued": {},
+                "dependencies_invoked": {},
```

Comment:
we still consider the `node_executrion_cache` to be an unstable API, and we should feel free to mess with it